include/openssl/srp.h.in

```diff
@@ -39,6 +39,8 @@ use OpenSSL::stackhash qw(generate_stack_macros);
 extern "C" {
 # endif
 
+# ifndef OPENSSL_NO_DEPRECATED_3_0
```

Comment:
Or actually when we have the disable cascade to OPENSSL_NO_SRP, perhaps we do not actually need this ifndef?

Reply:
I think we need it to support "Configure --api=1.1.1 no-deprecated"